demo/src/Menu.svelte

```diff
@@ -8,7 +8,7 @@
   <span slot="title">Examples</span>
 
   <ul>
-    {#each examples as example}
+    {#each examples.sort((a, b) => a.index - b.index) as example}
```

Comment:
This fixes the examples' incorrect order. Didn't notice it in time because it's only now when I added the 10th example. And without this fix the 10th example appears before the 2nd example. Like 10 < 2, 22 < 3 and so on.

Reply:
Nice